modules/nf-core/hamronization/abricate/main.nf

```diff
@@ -34,6 +34,25 @@ process HAMRONIZATION_ABRICATE {
         --reference_database_version ${reference_db_version} \\
         > ${prefix}.${format}
 
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        hamronization: \$(echo \$(hamronize --version 2>&1) | cut -f 2 -d ' ' )
+    END_VERSIONS
+    """
+
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    echo "hamronize \\
+        abricate \\
+        ${report} \\
+        $args \\
+        --format ${format} \\
+        --analysis_software_version ${software_version} \\
+        --reference_database_version ${reference_db_version} \\
+        > ${prefix}.${format}"
+
+    touch ${prefix}.${format}
```

Comment:
no need to both `echo` and `touch` here, I think you can pick one

Reply:
Alright, removed!